dapp/src/components/dashboard/components/Bridge/index.jsx

```diff
@@ -67,7 +67,7 @@ class Bridge extends Component {
 
     loadModal(SHOW_MORE_MODAL, {
       name: convertNetworkName(bridgeStatus[side].network),
-      network: bridgeStatus[side].network !== 'fuse' ? `https://api.infura.io/v1/jsonrpc/${bridgeStatus[side].network}` : 'https://rpc.fusenet.io',
+      network: bridgeStatus[side].network !== 'fuse' ? `https://api.infura.io/v1/jsonrpc/${bridgeStatus[side].network}` : 'https://rpc.fuse.io',
```

Comment:
hey can you use config var for fuse rpc? Also what's `https://api.infura.io/v1/jsonrpc/..` URL?

Reply:
the url represent the RPC Network that we show in the Show more modal.